src/ucp/api/ucp.h

```diff
@@ -333,6 +335,19 @@ typedef enum {
 
 /**
  * @ingroup UCP_DATATYPE
+ * @brief Generate a reusable identifier for Scatter-gather IOV data type.
```

Comment:
Now I understand your suggestion.

There are two approaches here: one is to pass "is_reusable" param on creation (to any datatype), and the other is to have separate datatypes for "reusable" and "non-reusable". IMO we want the same approach to all datatypes and modes. IOV never creates a handle, and strided always does, so i think If we want "is_reusable" as a parameter we should have IOV create a handle (needs to be regardless of "is_reusable", and redundant if it's false) - but not sure it's good for performance. Do you think it's ok to leave IOV reusable and non-reusable separate, but have a unified strided datatype?

(the current proposal for "reusable" IOV used create_dt, but will need to be part of create_iov_dt after moving from macros/varargs to functions)